infrastructure/template.yaml

```diff
@@ -411,34 +411,91 @@ Resources:
         #         - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${CommonStackName}/SessionTableName"
         #         - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${CommonStackName}/SessionTtl"
         #         - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${CommonStackName}/verifiable-credential/issuer"
-        - Statement:
-            - Effect: Allow
-              Action:
-                - ssm:GetParametersByPath
-              Resource:
-                - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/clients/*"
+  #       - Statement:
+  #           - Effect: Allow
+  #             Action:
+  #               - ssm:GetParametersByPath
+  #             Resource:
+  #               - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/clients/*"
+  #   Metadata: # Manage esbuild properties
+  #     BuildMethod: esbuild
+  #     BuildProperties:
+  #       Minify: true
+  #       Target: "es2020"
+  #       Sourcemap: true # Enabling source maps will create the required NODE_OPTIONS environment variables on your lambda function during sam build
+  #       EntryPoints:
+  #       - app.ts  
+
+  # HelloWorldFunctionLogGroup:
+  #   Type: AWS::Logs::LogGroup
+  #   Properties:
+  #     LogGroupName: !Sub "/aws/lambda/${HelloWorldFunction}"
+  #     RetentionInDays: 30
+
+  # HelloWorldFunctionLogsSubscriptionFilterCSLS:
+  #   Type: AWS::Logs::SubscriptionFilter
+  #   Condition: IsNotDevEnvironment
+  #   Properties:
+  #     DestinationArn: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython"
+  #     FilterPattern: ""
+  #     LogGroupName: !Ref HelloWorldFunctionLogGroup
+
+ ####################################################################
+#                                                                  #
+# Claimed Identity Function                                        #
+#                                                                  #
+####################################################################
+ 
+ ClaimedIdentityFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: app.lambdaHandler
+      CodeUri: ../src/claimed-identity/
+      Runtime: nodejs18.x
+      Environment:
+        Variables:
+          SESSION_TABLE: !Ref SessionTable
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSXrayWriteOnlyAccess
+        - DynamoDBWritePolicy:
+            TableName: !Sub "{{resolve:ssm:/${CommonStackName}/SessionTableName}}"
+        - DynamoDBReadPolicy:
+            TableName: !Sub "{{resolve:ssm:/${CommonStackName}/SessionTableName}}"
+        - DynamoDBReadPolicy:
```

Comment:
this one doesn't need access to PersonIdentityTable

Reply:
could screenshot be added to the ticket pls for dev testing

Reply:
commit with latest changes. It wont go pass the pipeline as its looking for CI lambda. Which screenshot are you referring?